CONTRIBUTING.md

```diff
@@ -0,0 +1,5 @@
+# Contributing to kubevirt/project-infra
+
+Welcome! As stated in the [README](README.md) this repository provides supporting code for the project infrastructure.
+
+See [the KubeVirt contribution guide](https://github.com/kubevirt/kubevirt/blob/master/CONTRIBUTING.md) for general information about how to contribute.
```

Comment:
We also point to this file with our DCO check. Might make sense to put a link to our license here too.

Reply:
Good point!

Reply:
Hm, IDK if we need another pointer towards our license as this file points to the README already which contains the details of licensing? Or should we add another direct reference to the license here? I searched for linking directly to the paragraph in the README but that doesn't seem to be supported. And I don't want to repeat myself also?